flink-queryable-state/flink-queryable-state-client-java/src/main/java/org/apache/flink/queryablestate/client/QueryableStateClient.java

```diff
@@ -67,6 +79,21 @@
 
 	private static final Logger LOG = LoggerFactory.getLogger(QueryableStateClient.class);
 
+	private static final Map<Class<? extends StateDescriptor>, StateFactory> STATE_FACTORIES =
+		org.apache.flink.shaded.guava18.com.google.common.collect.ImmutableMap
```

Comment:
We have unmodifiable map alternatives in the JDK. If there is not a good reason why we need guava here, I would suggest to solve this without this import. Even in that case, I would use import and not fully qualified classname.